### PR TITLE
View only by space

### PR DESCRIPTION
**Pull request by alexwojtala**
## Issue
Resolves 3 57

## What was done
- [x] Add read only toggle at the space level to allow spaces to individually turn on and off read only

## How to test
1. Use the toggle to turn on read only in share access modal
2. Make sure someone without access to space has read only view
3. Use toggle to turn off read only
4. Make sure that same person now does not have read only access to the space

### CODE REVIEW

File: ui/src/AccountDropdown/EditContributorsForm.scss [hunk 1]
```diff
@@ -65,5 +63,124 @@
       }
     }
   }
+  .inviteViewersLabel{
```
Comment (aclift19Ford) [inline ui/src/AccountDropdown/EditContributorsForm.scss:66]: I'd leave lots of comments about formatting and using elements vs classes in this file, but Elise and I are on a ticket that will rework a lot of this as soon as it gets merged in, so we can clean it up.

**Review by egriff38ford — APPROVED** [review]
🚢 🇮🇹 

Lots of cleanup for us to do, but looks good to us

**Review by smle225 — APPROVED** [review]
Still need to change ui elements

**Comment by smle225** [comment]
Header (Invite others to view)
- Change the verbiage: Share access --> Invite others to view 
- Left aligned
- Add collapsible/expandable icons

People with this link...
- Change the font color #403D3D

View only access is disabled
- Change the font color #403D3D
- Font weight: 700 (Bold)
- i icon increase the size to 14 x14
- tooltip popup font style - helvetica (it's roboto now)

Invite others to edit
- Add collapsible/expandable icons

People with this permission can edit
- Font size 16px
- Possibly a new story for adding a cdsid to design like below
Screen Shot 2021-01-20 at 2 05 35 PM

**Pull request merged**